media-gfx/phototonic/phototonic-2.1.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="An image viewer and organizer built with Qt and Exiv2"
+DESCRIPTION="Features
+* Support for common image formats and GIF animation
+* Supports tagging images, and filtering images by tags (IPTC)
+* Browse thumbnails recursively down a folder tree
+* Dynamic thumbnails loading
+* Image transformation and color manipulation
+* Display image information and metadata
+* Does not depend on any desktop environment."
+HOMEPAGE="https://github.com/oferkv/phototonic/"
+COPYRIGHT="2015-2018 Over Kashayov
+           2015-2018 Christopher roy
+           2015-2018 Krzysztof Pyrkosz
```

Comment:
```diff
-HOMEPAGE="https://github.com/oferkv/phototonic/"
+HOMEPAGE="https://github.com/oferkv/phototonic"
 COPYRIGHT="2015-2018 Over Kashayov
-           2015-2018 Christopher roy
-           2015-2018 Krzysztof Pyrkosz
-           2015-2018 Roman Chistokhodov
-           2015-2018 Thomas Lübking
-           2015-2018 Tung Le"
+	2015-2018 Christopher Roy
+	2015-2018 Krzysztof Pyrkosz
+	2015-2018 Roman Chistokhodov
+	2015-2018 Thomas Lübking
+	2015-2018 Tung Le"
 LICENSE="GNU GPL v3"
```
(In lines 12 to 16 we need to have 1 tab instead of 11 white spaces.)
(Please also remove the trailing slash in [`HOMEPAGE`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#homepage).)

BTW, I did not test the recipe yet but the rest looks OK.

Reply:
@fbrosson  fixed.. thanks.. i'm tested on x86_64